lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/src/string_helpers.cpp

```diff
@@ -25,9 +25,9 @@
 #include <codecvt>
 #include <ctime>
 #include <locale>
-#include <string_helpers.h>
 #include <sstream>
 #include <filesystem>
+#include "string_helpers.h"
```

Comment:
This was changed because when this particular file gets included as part of another binary, the angle bracket causes the linker to look in standard include directories instead of in ones that are common to the project - meaning when another project referenced these files and this .cpp was absorbed, it was unable to find the header. Changing it to be quotes resolves that issue by telling the linker to look inside the project.